client/src/components/PasswordInput.tsx

```diff
@@ -82,26 +82,41 @@ const PasswordInputWithoutI18n = forwardRef<HTMLInputElement, PasswordInputProps
             <LocaleLink
               to={`${account.forgotPassword}?email=${encodeURIComponent(username || "")}`}
             >
-              Forgot your password?
+              {i18n._(t`Forgot your password?`)}
             </LocaleLink>
           )}
         </div>
         {showPasswordRules && (
```

Comment:
thoughts on restructuring like this for more clarity?
```
showPasswordRules 
   ? <div className="password-input-rules"> 
   : {showError && error && <div className="password-input-errors">}
```

Reply:
oh yeah that's much better - thanks!